src/index.html

```diff
@@ -4,9 +4,344 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="nav_logo">
+        <img src="/src/images/logo.png" alt="logo moyo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="index.html" class="nav-link nav-link--is-active">Apple</a>
```

Comment:
```suggestion
            <a href="index.html" class="nav-link is-active">Apple</a>
```
Please rename it as it was in the Moyo Header task because tests work only with class `is-active` and not they can`t find this class, that's why your tests are failing now